src/views/virtualmachines/actions/components/VirtualMachineMigration/VirtualMachineMigrationModal.tsx

```diff
@@ -55,36 +53,29 @@ const VirtualMachineMigrateModal: FC<VirtualMachineMigrateModalProps> = ({
   vms,
 }) => {
   const { t } = useKubevirtTranslation();
+  const vmNamespaces = removeDuplicates(vms?.map((vm) => getNamespace(vm)));
 
-  const migrationNamespace = getNamespace(vms?.[0]);
   const cluster = getCluster(vms?.[0]);
```

Comment:
@pcbailey  @upalatucci 
it seems the VMs may be from different clusters

Reply:
we should not allow to migrate from different clusters. MigPlans do not manage migrations across different clusters